community/kernel/src/test/java/org/neo4j/kernel/impl/store/CommonAbstractStoreTest.java

```diff
@@ -193,7 +193,8 @@ public void recordCursorPinsEachPageItReads() throws Exception
     {
         File storeFile = dir.file( "a" );
         RecordingPageCacheTracer tracer = new RecordingPageCacheTracer( Pin.class );
-        PageCache pageCache = pageCacheRule.getPageCache( fileSystemRule.get(), tracer, Config.empty() );
+        PageCache pageCache = pageCacheRule.getPageCache( fileSystemRule.get(),
+                config().withTracer( tracer ), Config.empty() );
```

Comment:
Although it looks like [Config.empty()](https://github.com/neo4j/neo4j/blob/3.2/community/kernel/src/main/java/org/neo4j/kernel/configuration/Config.java#L87) still exists, and this rebased branch compiles and is generally happy.